projects/plugins/classic-theme-helper-plugin/classic-theme-helper-plugin.php

```diff
@@ -51,3 +51,20 @@ function ( $actions ) {
 		return $actions;
 	}
 );
+
+	// Init Jetpack packages
+	add_action( 'plugins_loaded', 'init_packages', 1 );
+
+	/**
+	 * Configure what Jetpack packages should get automatically initialized.
+	 *
+	 * @return void
+	 */
+function init_packages() {
+	if ( class_exists( 'Automattic\Jetpack\Classic_Theme_Helper\Main' ) ) {
```

Comment:
The class_exists check was added in here as it makes sense to have the check in case there are any package loading issues, but also because tests were failing (on deactivation and uninstallation of the plugin, as the class 'didn't exist').